src/validation.cpp

```diff
@@ -660,6 +667,10 @@ static bool AcceptToMemoryPoolWorker(const CChainParams& chainparams, CTxMemPool
             if (!pcoinsTip->HaveCoinInCache(txin.prevout)) {
                 coins_to_uncache.push_back(txin.prevout);
             }
+
+            // Note: this call incidentally adds txin.prevout to the coins cache
```

Comment:
Technically this only adds `txin.prevout` to the cache if it’s not already there (hence the call above to `HaveCoinInCache`).  Perhaps rephrase to “... this call may add txin.prevout...” ?

Reply:
Updated this, thanks!